src/assets/css/style.css

```diff
@@ -136,6 +136,13 @@ footer #version {
 /*
    CSS for styling content on PACER pages generated by the content script.
 */
+.recap-alert-banner {
```

Comment:
This seems repetitive, but I suppose it's fine. Someday we can refactor our CSS, I guess, if it ever gets unwieldy.